src/components/ArcGISWebMap.tsx

```diff
@@ -213,6 +213,11 @@ function FormElementArcGISWebMap({
 
         const layerList = new LayerList({
           view,
+          listItemCreatedFunction: (event) => {
+            if (element.readOnly && event.item.layer.title === 'Drawing') {
```

Comment:
on the incredibly small chance someone's map has a pre-configured layer with the title `Drawing`, this would hide it too. I think we're better off just not adding our drawing layer to the web map to begin with - have a squiz at line 182

Reply:
Ah, thought it was being generated automatically. Good call.